### PR TITLE
Lowercase/Uppercase String Transformers

### PR DESCRIPTION
**Pull request by mattruggio**
This PR adds two new transformers:

1. r/format/lowercase
2. r/format/uppercase

It also includes a fix so all code files have proper licensing commentary.

### CODE REVIEW

File: lib/realize/type/base.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Copyright (c) 2020-present, Blue Marble Payroll, LLC
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#
+
+module Realize
+  class Type
+    # Common code for all Type Transformer subclasses.
+    class Base
```
Comment (ddewar) [inline lib/realize/type/base.rb:13]: Is this going to be used with all transformers? Just so I know how to use this in the future, What's the goal of this?

File: lib/realize/type/string.rb [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+#
 
-      def initialize(nullable: false)
-        @nullable = nullable || false
-      end
+require_relative 'base'
 
+module Realize
+  class Type
+    # Call #to_s.upcase on the value and return result.
```
Comment (ddewar) [inline lib/realize/type/string.rb:14]: I don't think this comment goes to this class.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+# 1.4.0 (TBD)
```
Comment (ddewar) [inline CHANGELOG.md:1]: Still TBD?

**Review by ddewar — COMMENTED** [review]
Just some quick comments, code all looks good.

File: lib/realize/type/base.rb (hunk 1 above)
Comment (mattruggio) [reply lib/realize/type/base.rb:13]: It is used for all Type transformers within the Type namespace.

File: CHANGELOG.md (hunk 3 above)
Comment (mattruggio) [reply CHANGELOG.md:1]: Yes, dependent on approval

File: lib/realize/type/string.rb (hunk 2 above)
Comment (mattruggio) [reply lib/realize/type/string.rb:14]: ah! good call.

**Review by ddewar — APPROVED** [review]

**Pull request merged**